rtv/subreddit.py

```diff
@@ -73,6 +74,24 @@ def prompt_subreddit(self):
         if name is not None:
             self.refresh_content(order='ignore', name=name)
 
+    @SubredditController.register(Command('SUBREDDIT_FRONTPAGE'))
+    def show_frontpage(self):
```

Comment:
Hey, it looks like you're doing everything right. The refresh-token is the same one that's generated when you login to rtv. I think what's happening is that one of the tests in test_config.py is accidentally deleting the refresh-token from your ~/.config/rtv/ folder. That's not your fault and it's something that I need to fix. For now, try logging in again and copying the refresh-token to a different location. Then run the test again using the copied token, e.g.

```bash
$ cp ~/.config/rtv/refresh-token ~/refresh-token
$ PYTHONPATH=. py.test --record-mode=once --refresh-token=~/refresh-token
```

Let me know how it goes.